deploy/create_cert.sh

```diff
@@ -15,7 +15,7 @@
 # =============================================================================
 
 # Create certificate
-aws iot create-keys-and-certificate --set-as-active --certificate-pem-outfile "cert.pem" --public-key-outfile "public.key" --private-key-outfile "private.key" | tee cert_result.log
+aws iot create-keys-and-certificate --set-as-active --certificate-pem-outfile "cert.pem" --public-key-outfile "public.key" --private-key-outfile "private.key" | tee cert_result.log > /dev/null
```

Comment:
Is it a problem to output stdout? This line (using `tee`) is intentional to display the standard output of this command.